charts/stable/ring-mqtt/values.yaml

```diff
@@ -17,19 +25,20 @@ service:
         protocol: tcp
         targetPort: 8554
         port: 8554
-persistence:
-  data:
-    enabled: true
-    mountPath: /data
-portal:
-  open:
-    enabled: false
-securityContext:
-  container:
-    readOnlyRootFilesystem: false
-    runAsNonRoot: false
-    runAsGroup: 0
-    runAsUser: 0
+
+ring:
+  mqtt_url: "mqtt://localhost:1883"
+  mqtt_options: ""
+  livestream_user: ""
+  livestream_pass: ""
+  disarm_code: ""
+  enable_cameras: true
+  enable_modes: false
+  enable_panic: false
+  hass_topic: "homeassistant/status"
+  ring_topic: "ring"
+  location_ids": []
```

Comment:
It seems this is a list here, but on UI its string and toJson will make it string.